src/jukebox/components/playermpd/__init__.py

```diff
@@ -374,6 +504,25 @@ def initialize():
     volume = plugs.get('volume')
     volume.factory.register("mpd", MpdVolumeCtrlBuilder(player_ctrl))
 
+    # Update mpc library
+    library_update = cfg.setndefault('playermpd', 'library', 'update_on_startup', value=True)
+    if library_update:
+        player_ctrl.update()
+
+    # Check user rights: Where to get the music directory from? Should be mpd.conf, right?
+    audio_folder_path = None
+    mpd_conf_file = cfg.setndefault('playermpd', 'mpd_conf', value='/etc/mpd.conf')
```

Comment:
Are you reading `value` from the yaml file?
https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/resources/default-settings/jukebox.default.yaml#L47

Reply:
Yes, if the entry exists it is read from file. If the entry is missing it gets created with `value`